api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -34,7 +35,7 @@
 public class FreeTierBillingService {
 
   private final BigQueryService bigQueryService;
-  private final NotificationService notificationService;
```

Comment:
This was only used for free tier alert logging, and email alerting supersedes this.